packages/slice-machine/src/domain/customType.ts

```diff
@@ -425,6 +453,64 @@ export function reorderField(args: ReorderFieldArgs): CustomType {
   return newCustomType;
 }
 
+// if the UID is not existing in any section, it should be added to the main section
+export function addUIDField(label: string, customType: CustomType): CustomType {
+  const sectionId = getMainSectionId(customType);
+  if (typeof sectionId === "undefined") {
+    return customType;
+  }
+  const newFieldId = "uid";
+  const newField: UID = {
+    config: {
+      label,
+    },
+    type: "UID",
```

Comment:
💡 #idea

I would move the `type` above to keep current management with the type is first.

Reply:
I'm not sure if I understand 😅

Reply:
Today in the "json" type is the first property on top of config, I would keep it like this. No?

Reply:
ah ok, then I understood correctly, will change that in `DEFAULT_MAIN_WITH_UID` as well then